### DISCUSSION TITLE
I come from RVC and I have questions about so vits (pitch, intonations, configuration etc)

### DISCUSSION THREAD

**Discussion by GPU-server** [voicepaw/so-vits-svc-fork#1238] [Q&A]
Hello, with RVC I was able to create / train modes.pth,
But I don't know much about vits;
1) can we train models?
2) can we influence all of vocie timber, intonation, pitch etc?
In rvc I did know much about that so I am searching for a tool where I can have more control.
Thanks

**Comment by 34j** [comment]
1. Probably yes
2. not very much
